app/containers/message/Components/CollapsibleQuote.tsx

```diff
@@ -0,0 +1,183 @@
+import { transparentize } from 'color2k';
+import { dequal } from 'dequal';
+import React, { useContext, useState } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+import { themes } from '../../../constants/colors';
+import { IAttachment } from '../../../definitions/IAttachment';
+import { TGetCustomEmoji } from '../../../definitions/IEmoji';
+import { CustomIcon } from '../../../lib/Icons';
+import sharedStyles from '../../../views/Styles';
+import Markdown from '../../markdown';
+import MessageContext from '../Context';
+import Touchable from '../Touchable';
+
+const styles = StyleSheet.create({
+	button: {
+		flex: 1,
+		flexDirection: 'row',
+		alignItems: 'center',
+		marginTop: 6,
+		alignSelf: 'flex-start',
+		borderWidth: 1,
+		borderRadius: 4
+	},
+	attachmentContainer: {
+		flex: 1,
+		borderRadius: 4,
+		flexDirection: 'column',
+		padding: 8
+	},
+	authorContainer: {
+		flex: 1,
+		flexDirection: 'row',
+		alignItems: 'center'
+	},
+	fieldsContainer: {
+		flex: 1,
+		flexWrap: 'wrap',
+		flexDirection: 'row'
+	},
+	fieldContainer: {
+		flexDirection: 'column',
+		padding: 10
+	},
+	fieldTitle: {
+		fontSize: 15,
+		...sharedStyles.textBold
+	},
+	marginTop: {
+		marginTop: 4
+	},
+	marginBottom: {
+		marginBottom: 4
+	},
+	title: {
+		flex: 1,
+		fontSize: 16,
+		...sharedStyles.textMedium
+	},
+	touchableContainer: { flex: 1, flexDirection: 'row' },
```

Comment:
I think `format on save` isn't configured on your env.
These styles should have been formatted as the ones above.